infermedica_api/webservice.py

```diff
@@ -238,14 +238,21 @@ def suggest(self, diagnosis_request, max_results=8, interview_id=None):
         """
         method = self.__get_method('suggest')
         headers = self.__get_interview_id_headers(interview_id=interview_id)
+        params = kwargs
```

Comment:
Either this or [this](https://github.com/infermedica/python-api/pull/8/files#r471714248)?

Reply:
Same here `params = kwargs.pop('params', {})`

Reply:
@akszydelko done